locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -82,64 +82,44 @@ msgstr ""
 "er nicht unseren Diskussionsregeln (Netiquette) entspricht."
 
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
```

Comment:
@goapunk there are a lot of deleted translation ref files?!

Reply:
all those changes are coming from `make po`, or is there anything else which needs to be done?

Reply:
no but we have had issues in the past where js file references are deleted, we do still need the references though, I will try running make po on your PR and check

Reply:
@goapunk I pushed mine which brings it all back, shall I just merge the commits?

Reply:
yeah, thanks.